src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionUpToDateChecker.cs

```diff
@@ -81,14 +100,15 @@ public IEnumerable<string> PerformUpToDateCheck(DependencyGraphSpec dependencyGr
                     {
                         dirtySpecs.Add(projectUniqueName);
                     }
```

Comment:
Space here.

Reply:
Nothing that says we should add extra spaces there. 
This is actually old code too, maybe you intended to comment on a separate line?